src/App.js

```diff
@@ -1,25 +1,63 @@
 import React from 'react';
 
 import './App.scss';
+import Clock from './Components/Clock';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: Math.random(),
+    clockNewName: Math.random(),
```

Comment:
```suggestion
    clockNewName: null,
```
initially, you don't need  newName